style.css

```diff
@@ -0,0 +1,142 @@
+.app-bar {
+  display: flex;
+  background-color: #3c3a39;
+  width: 375px;
+  height: 92px;
+  position: absolute;
+  margin: 0;
+}
+
+.logo-placeholder {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  position: absolute;
+  width: 94px;
+  height: 44px;
+  left: 12px;
+  top: calc(50% - 44px / 2);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.menu-container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.menu {
+  height: 40px;
+  width: 40px;
+  margin-left: 0;
+  margin-top: 0;
+  border-radius:  0px;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  position: absolute;
+  height: 590px;
+  width: 375px;
+  top: 92px;
+  padding: 120px 24px 120px 24px;
+  gap: 12px;
+  background: #1c1a19;
+}
+
+.hello {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  color: #ff6b00;
+  letter-spacing: 0.37px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.image {
+  position: absolute;
+  width: 406.89px;
+  height: 312.03px;
+  left: 230.1px;
+  top: 190px;
+}
+
+.john {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  height: 27px;
+  width: 327px;
+  color: #fff5e1;
+  display: flex;
+  align-items: center;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.dText {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 123px;
+  color: #fff;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.SM-Container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 200px;
+  top: 430px;
+  left: 24px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.box {
+  width: 100px;
+}
```

Comment:
- [ ] Please ensure that your project is responsive without any horizontal overflow at the right side to make your project look good on mobile devices. Secondly, your social media icons are not aligned in the right place as it is in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) so make sure you adjust your social media icons to make them looks exactly as the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). :pray: See the addressed issue in the video below :point_down: 
  
  [Screencast from 06-02-2023 10:33:29 AM.webm](https://github.com/JohnteFresh/Mobile-Portfolio-setup/assets/92303891/8e8f6bae-2dac-4751-88dd-802f9d197ffb)